.gitignore

```diff
@@ -6,6 +6,7 @@ dist/
 
 # Test files
 .nox/
+venv/
```

Comment:
`venv` is likely the name of your specific virtual environment. Can we revert this change?

Reply:
whoops, thanks for spotting that!